hieradata_aws/class/staging/db_admin.yaml

```
@@ -144,6 +144,17 @@ govuk_env_sync::tasks:
     temppath: "/tmp/whitehall_production"
     url: "govuk-production-database-backups"
     path: "mysql"
+  "pull_release_staging_daily":
+    ensure: "present"
+    hour: "2"
+    minute: "15"
+    action: "pull"
+    dbms: "mysql"
+    storagebackend: "s3"
+    database: "release_production"
+    temppath: "/tmp/release_production"
+    url: "govuk-staging-database-backups"
```

This needs to download the Prod backup, right?
```suggestion
    url: "govuk-production-database-backups"
```

Maybe also update the title to "pull_release_production_daily" since that seems to match the surrounding ones? \shrug

---

AGREED. I looked at the wrong file. This will be rolled in to my next changes